com.microsoft.mrtk.input/Visualizers/Visuals/ControllerModelLoader.cs

```diff
@@ -0,0 +1,64 @@
+using System.Collections.Generic;
+using UnityEngine;
+using Microsoft.MixedReality.OpenXR;
+using System.Threading.Tasks;
+
+#if GLTFAST_PRESENT
+using GLTFast;
+#endif
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    public class ControllerModelLoader
+    {
+        public static Dictionary<ulong, GameObject> ControllerModelDictionary;
+
+        public async static Task<GameObject> TryGenerateControllerModelFromPlatformSDK(ControllerModel controllerModelProvider)
+        {
+            GameObject gltfGameObject = null;
+
+#if MROPENXR_PRESENT && (UNITY_STANDALONE_WIN || UNITY_WSA || UNITY_ANDROID) && GLTFAST_PRESENT
```

Comment:
Why all these platform ifdefs?

Reply:
These flags were required to prevent the MR Open XR plugin from breaking builds on other platforms like WebGL.